packages/grid/_modules_/grid/models/params/gridColumnHeaderParams.ts

```diff
@@ -5,7 +5,7 @@ export interface GridColumnHeaderParams {
   /**
    * The HTMLElement column header element.
    */
-  element?: HTMLElement | null;
+  getElement: (field: string) => HTMLElement | null;
```

Comment:
This doesn't make sense, we are already on a specific field
```suggestion
  getElement: () => HTMLElement | null;
```